README.md

```diff
@@ -162,6 +191,8 @@ queued.  The 5th and 6th B's are already interesting on their own: they won't be
 are only three active jobs; the third A won't be started until *either* there are only three active
 jobs, or one of the other A's finish.
 
+[//]: # s/jobs/threads ?
```

Comment:
They are distinct: each job gets processed by a thread, but there can be waiting jobs because there are no free worker threads to process them yet.